batchflow/models/torch/blocks/named_blocks.py

```diff
@@ -120,23 +112,77 @@ def __init__(self, inputs=None, layout='cnacn', channels='same', kernel_size=3,
-                         'branch/stride': branch_stride_downsample}]
+                         'branch': branch_params}]
         layer_params += [{}]*(n_reps-1)
 
         super().__init__(*layer_params, inputs=inputs, layout=layout, channels=channels,
                          kernel_size=kernel_size, stride=stride, groups=groups, attention=attention,
                          **kwargs)
 
 
+class BottleneckBlock(ResBlock):
+    """ ResBlock with a canonical bottleneck (1x1 conv-batchnorm-activation) architecture.
+
+    Parameters
+    ----------
+    inputs : torch.Tensor
+        Example of input tensor to this layer.
+    layout : str
+        A sequence of letters, each letter meaning individual operation.
+        See more in :class:`~.layers.conv_block.BaseBlock` documentation. Default is 'cna cna cn'.
+    channels : int, str, list of int, list of str
+        If `str`, then number of channels is calculated by its evaluation. ``'S'`` and ``'same'`` stand for the
+        number of channels in the previous tensor. Note the `eval` usage under the hood.
+        If int, then number of channels in the output tensor. Default value is 'same'.
+    kernel_size : int, list of int
+        Convolution kernel size. Default is [1, 3, 1].
+    bottleneck : int
+        Factor of channels increase. Default is 4.
+    expand_channels : bool
+        If False, channels are increased in the first convolution layer only. Default is False.
+        If True, channels are increased in all but the last convolution layers.
+    groups : int, list of int
+        Use `groups` convolution side by side, each  seeing 1 / `groups` the input channels,
+        and producing 1 / `groups` the output channels, and both subsequently concatenated.
+        Number of `inputs` channels must be divisible by `groups`. Default is 1.
+    expand_groups : bool
+        If True and `groups` is int, `groups` is propagated to all convolution layers.
+        If True and `groups` is list of int, nothing is changed.
+        If False, `groups` is set to 1 in the first and the last 1x1 convolution layers. Default is False.
+    kwargs : dict
+        Other named arguments for the :class:`~.layers.ResBlock`
+    """
+    def __init__(self, inputs=None, layout='cna cna cn', channels='same', kernel_size=(1, 3, 1),
+                 bottleneck=4, expand_channels=False, expand_groups=False, groups=1, **kwargs):
+        num_convs = sum(letter in CONV_LETTERS for letter in layout)
+
+        channels = [channels] * num_convs if isinstance(channels, (int, str)) else channels
+        channels = [safe_eval(item, get_num_channels(inputs)) if isinstance(item, str) else item
+                   for item in channels]
+        if expand_channels:
+            channels = [item * bottleneck for item in channels[:-1]] + [channels[-1]]
+        else:
+            channels[0] *= bottleneck
+
+        kernel_size = [kernel_size] * num_convs if isinstance(kernel_size, int) else list(kernel_size)
```

Comment:
Why `list(kernel_size)`? Can we use the `kernel_size` instead of this?

Reply:
`Block` needs it as `list`